pkg/config/config.go

```diff
@@ -130,21 +132,48 @@ func (c *ConfigInfo) writeToFile() error {
 }
 
 // SetConfiguration modifies Odo configurations in the config file
-// as of now only being used for updatenotification
-func (c *ConfigInfo) SetConfiguration(parameter string, value bool) error {
+// as of now being used for timeout, updatenotification
+func (c *ConfigInfo) SetConfiguration(parameter string, value interface{}) error {
 	switch parameter {
+
+	case "timeout":
+		typedval, ok := value.(int)
+		if typedval == 0 {
+			fmt.Println("Cannot set timeout to 0, minimum value is 1")
+			return errors.Errorf("type assertion error: unable to set")
```

Comment:
I think error should be: "Cannot set timeout to 0, minimum value is 1" instead of typeassertion error

Reply:
ack